### PR TITLE
Fixing misc warnings in the console

### PR DESCRIPTION
**Pull request by ash3rz**

### CODE REVIEW

**Review by ianmcorvidae — APPROVED** [review]
gotta love that sweet sweet "make the console stop yelling about things" project

File: src/components/layout/UserPortalUpdatePrompts.js [hunk 1]
```diff
@@ -140,7 +140,6 @@ function UserPortalUpdatePrompts() {
                 open={!!userPortalResp?.update_required}
                 id={baseId}
                 fullWidth
-                disableBackdropClick
```
Comment (psarando) [inline src/components/layout/UserPortalUpdatePrompts.js:143]: Part of the deprecation warning for this prop says:
> Use the onClose prop with the `reason` argument to filter the `backdropClick` events.

We should test this dialog to ensure that users can not close it by clicking outside of it.

**Review by psarando — APPROVED** [review]
LGTM 👍 ✨ 

Though we should ensure that users can't close the dialog after removing the `disableBackdropClick` prop.

**Review by sri2k1us — APPROVED** [review]
:shipit:

File: src/components/layout/UserPortalUpdatePrompts.js (hunk 1 above)
Comment (ash3rz) [reply src/components/layout/UserPortalUpdatePrompts.js:143]: Good catch 👍 I did test it within FireFox, Chrome, and Safari and the dialog doesn't close when clicking outside of it.  I think having `open` based on the user portal response and not having an `onClose` defined is probably preventing it from closing without that explicit setting.

**Comment by ash3rz** [comment]
Thanks for the reviews!

**Pull request merged**